Changes

```diff
@@ -194,6 +194,8 @@ Compilers:
   (Jérôme Vouillon)
 
 Runtime system:
+- PR#7173 GPR#515: Add a public C API for weak arrays and ephemerons.
+  (François Bobot and Jacques-Henri Jourdan, review by Mark Shinwell)
```

Comment:
I suspect you broke the careful sorting of the issue list for this and `GPR#22` just to spite me :-)